circuit_puzzles/include/oracle.clib

```diff
@@ -1,12 +1,17 @@
 (
 
-  (defun cut-prices (price_list expiration_timestamp)
+  (defun cut-prices (price_list expiration_timestamp last_matured_price_info)
```

Comment:
add comment to function that it returns (matured_price_info matured_price_info (list non-matured price infos)) if there is a matured_price_info, or (() (list non-matured price infos)) otherwise, as one wouldn't necessarily expect this